src/js/utils/NewsAPI.js

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+
+export default {
+  getSources: (url) => {
+    return new Promise((resolve, reject) => {
+      axios.get(url)
+      .then((res) => {
+        resolve((res.data.sources));
+      });
+      // .catch((error) => {
+      //   if (error) reject(error);
+      // });
+    });
```

Comment:
Why do you have the errors commented out? You should ideally be catching those errors and managing or processing them